README.md

```diff
@@ -1,2 +1,49 @@
 # form-gen-tool
-Dynamically create HTML forms
+
+*Dynamically create HTML forms*
+
+## Setup 
+
+### Dependencies
+
+* node
+* npm
+
+### Installation
+
+* Clone this repo:
+git@github.com:cayouetten/form-gen-tool.git
+
+## Test
+
+* cd form-gen-tool
+* node ./src/index.js
+
+### Getting started
+
+* npm install
+
+## Usage
+
+Enter array of input data objects and render an HTML formatted form
+* Supports types: label, text, email, password, submit
+* Supports attributes: name, id, class, value, required
+
+## Demo
```

Comment:
I don't think you need a "Demo" heading below "Usage" if you provide a lead-in sentence and wrap the code example below in triple-backticks.